build-ppc64le.patch

```diff
@@ -44,36 +44,58 @@ index f8226e1..dc9980f 100755
    *)
      exit 1 ;;
  esac
+diff --git a/images/local-path-provisioner/Dockerfile b/images/local-path-provisioner/Dockerfile
+index df601536..e2050319 100644
+--- a/images/local-path-provisioner/Dockerfile
++++ b/images/local-path-provisioner/Dockerfile
+@@ -15,7 +15,7 @@
+ # NOTE the actual go version will be overridden
+ FROM --platform=$BUILDPLATFORM docker.io/library/golang:latest
+ COPY --chmod=0755 scripts/third_party/gimme/gimme /usr/local/bin/
+-RUN git clone --filter=tree:0 https://github.com/rancher/local-path-provisioner
```

Comment:
Since local-path-provisioner is not available on ppc64le, a PR to add support to enable building on power has been raised.